src/models/logs.ts

```diff
@@ -1,32 +1,45 @@
-const config = require('config');
-const fs = require('fs');
-const path = require('path');
-const sqlite3 = require('sqlite3').verbose();
+import config from 'config';
+import fs from 'fs';
+import path from 'path';
+import {verbose, Database as SQLiteDatabase} from 'sqlite3';
+import * as Constants from '../constants';
+import UserProfile from '../user-profile';
 
 const AddonManager = require('../addon-manager');
-const Constants = require('../constants');
-const UserProfile = require('../user-profile').default;
+
+const sqlite3 = verbose();
 
 const METRICS_NUMBER = 'metricsNumber';
 const METRICS_BOOLEAN = 'metricsBoolean';
 const METRICS_OTHER = 'metricsOther';
 
 class Logs {
+  private db: SQLiteDatabase|null;
+
+  private idToDescr: {[id: number]: any};
+
+  private descrToId: {[descr: string]: number};
```

Comment:
You could use `Record` here.
```suggestion
  private descrToId: Record<string, number>;
```